spec/models/generic_file_spec.rb

```diff
@@ -1,7 +1,35 @@
 require 'rails_helper'
 
-RSpec.describe GenericFile do
-  it "should be able to persist" do
-    expect{GenericFile.new.save}.not_to raise_error
+RSpec.describe GenericFile do  
```

Comment:
I think this test should be placed in enricher_spec instead of generic_file_spec.